### PR TITLE
napadaan lang po, might help

### PR DESCRIPTION
**Pull request by lightshire**
- not sure if will work
 - just got curious with the repo.

### CODE REVIEW

**Comment by lightshire** [comment]
might be interested in adding a switch-case rather than if?

**Comment by wildtron** [comment]
:+1:

File: js/streamers.js [hunk 1]
```diff
@@ -94,22 +94,25 @@
...
+                            img = 'images/noprevhb.jpg';
+                        }
+                        if (platform === 'Dailymotion') {
+                            img = 'images/noprevdm.jpg';
+                        }
+                        if (!platform === 'YouTube Gaming') {
+                            img = 'images/noprevgy.jpg';
+                        }
                     }
+                    bg = <img className='lazy stream_preview' src={img} alt={status} title={status}/>
```
Comment (nhvu1988) [inline js/streamers.js]:
```js
process_stream_preview = function (platform, preview, status) {
    var img = preview || '';

    if (!img) {
        switch (platform) {
            ...
        }
    }

    return <img className='lazy stream_preview' src={img} alt={status} title={status}/>
},
```

Comment (lightshire) [inline js/streamers.js]: ^better

**Comment by lightshire** [comment]
check new patch, added switch case

**Pull request merged**